src/runtime_src/core/tools/xbmgmt2/flash/xqspips.cpp

```diff
@@ -24,6 +24,7 @@
 #include "core/common/query_requests.h"
```

Comment:
Copyright date (line 2) needs to be updated.

Reply:
comment addressed.